src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -29,6 +18,21 @@ or by adding the following element to the project file:
   </ItemGroup>
 ```
 
+## Using daily builds
+
+For using daily builds, you need to make sure the `nuget.config` file for your project contains the following package sources under the `<packageSources>` element:
```

Comment:
Adding a nuget feed alone is not sufficient to give you daily builds with the SDK integration.

It needs to something like https://github.com/dotnet/runtime/blob/main/docs/project/dogfooding.md#install-prerequisites . Maybe just refer to that doc.

Reply:
Based on my experiments to use latest ILC with Preview 5 for example, I have to add following custom target.
```
<Target Name="ForceNightlyILC" BeforeTargets="ProcessFrameworkReferences">
  <ItemGroup>
    <KnownILCompilerPack Update="Microsoft.DotNet.ILCompiler">
      <TargetFramework>net7.0</TargetFramework>
      <ILCompilerPackNamePattern>runtime.**RID**.Microsoft.DotNet.ILCompiler</ILCompilerPackNamePattern>
      <ILCompilerPackVersion>7.0.0-preview.6.22325.1</ILCompilerPackVersion>
      <ILCompilerRuntimeIdentifiers>linux-musl-x64;linux-x64;win-x64;linux-arm;win-arm64;linux-arm64</ILCompilerRuntimeIdentifiers>
    </KnownILCompilerPack>
  </ItemGroup>
</Target>
```

Simply changing `RuntimeFrameworkVersion` was not working as planned.

Reply:
Hmm, changing `RuntimeFrameworkVersion` is not sufficient to publish regular self-contained apps either. The steps in https://github.com/dotnet/runtime/blob/main/docs/project/dogfooding.md#option-2-self-contained do not work.

The problem is that changing `RuntimeFrameworkVersion` also sets the version for Microsoft.AspNetCore.App.Runtime and Microsoft.WindowsDesktop.App.Runtime. The version numbers of nightly builds of all runtime packs rarely match.

I guess we should keep recommending to use the classic package reference (`<PackageReference Include="Microsoft.DotNet.ILCompiler" Version="7.0.0-*" />`) to get the nightly builds.